datamodel/app/view/vw_tww_wastewater_structure.py

```diff
@@ -262,12 +264,25 @@ def vw_tww_wastewater_structure(srid: int, pg_service: str = None, extra_definit
         SET fk_main_wastewater_node = NEW.wn_obj_id
         WHERE obj_id = NEW.obj_id;
 
-    {insert_vw_cover}
+      -- Aggregate all co_* values in a jsonb
+      new_co := (
+          SELECT jsonb_object_agg(key, to_jsonb(NEW)->key)
+          FROM jsonb_object_keys(to_jsonb(NEW)) key
+          WHERE LEFT(key, 3) = 'co_' AND key NOT IN ('co_identifier', 'co_obj_id')
+        );
+
+      -- Check if all remaining values are NULL
+      CASE WHEN jsonb_strip_nulls(new_co)::text <> '' THEN -- no cover entries
+        {insert_vw_cover}
 
-      UPDATE tww_od.wastewater_structure
+       UPDATE tww_od.wastewater_structure
         SET fk_main_cover = NEW.co_obj_id
         WHERE obj_id = NEW.obj_id;
 
+     ELSE
+         RAISE WARNING 'Object_ID %: no cover created as all cover-related columns are NULL', NEW.obj_id; -- Warning
```

Comment:
@cymed How does this look like in TEKSI? Can you add a screen shot?
Is 'Object_ID% a helpful text? We never used `Object_ID` anywhere; the attributes name is obj_id or co_obj_id. 
Maybe add: If you want to add a cover please fill in at least one cover attribute value.

Reply:
In QGIS, the warning is shown in the log window: 
![grafik](https://github.com/teksi/wastewater/assets/102035587/d5d89add-9cfd-42a2-98b4-f5ed189ee969)
https://github.com/teksi/wastewater/pull/36/commits/47bf16b46502f1ceacd67cb12f0cf53a11b527f4 alters the warning text